packages/components/src/CircularProgress/CircularProgress.component.js

```diff
@@ -38,8 +40,19 @@ function CircularProgress({ size, light, percent, className }) {
 		[theme.large]: size === SIZE.large,
 	});
 
+	const percentLabel = percent && `${percent}%`;
 	return (
-		<svg focusable="false" className={classes} viewBox={`0 0 ${DIAMETER} ${DIAMETER}`}>
+		<svg
+			focusable="false"
+			className={classes}
+			viewBox={`0 0 ${DIAMETER} ${DIAMETER}`}
+			aria-busy="true"
+			aria-live="polite"
```

Comment:
Why do you force it to be read again for each modification?

Reply:
This is not something like errors where we need to break the user flow with an important info.
Here a progress can take some time and be updated `very` regularly.

While the "thing" is loading, if the rest of the application is not blocked, user will be interrupted frequently in his actions with the percentage info, which is disturbing.

In the case where the rest of the application is blocked, it's ok, the user will be idle, and the info will be read at each update.

Reply:
Ok, after a little chat, I didn't understand the first message.
`aria-live` removed, thanks